OneSignalSDK/onesignal/src/main/java/com/onesignal/GooglePlayServicesUpgradePrompt.java

```diff
@@ -39,7 +28,7 @@ private static boolean isGooglePlayStoreInstalled() {
    }
 
    static void ShowUpdateGPSDialog() {
```

Comment:
Convert `ShowUpdateGPSDialog` to lowercase first letter? 

`showUpdateGPSDialog`

Reply:
👍 Good catch, might as well clean this up while in here